src/Hidden/HiddenCss.spec.js

```diff
@@ -25,7 +27,11 @@ describe('<HiddenCss />', () => {
           <div className="foo" />
         </HiddenCss>,
       );
-      assert.strictEqual(wrapper.props().className, `foo ${classes.onlySm}`);
+      assert.strictEqual(wrapper.type(), 'span');
+      assert.isTrue(wrapper.hasClass(classes.onlySm));
```

Comment:
What do you think of sticking to `assert.strictEqual(X, true)`? I have found two other tests file using this pattern.  It's almost not used.